includes/ucf-post-list-config.php

```diff
@@ -212,6 +221,9 @@ public static function get_options() {
 
 				// Tag
 				'tag'         => new UCF_Post_List_Option( 'tag' ),
+				'tag_name'    => new UCF_Post_List_Option( 'tag_name' , array(
```

Comment:
Can this `tag_name` param be removed?  `tag_name` doesn't appear to be a supported parameter for `WP_Query`, and would need additional custom logic to work as intended.  https://developer.wordpress.org/reference/classes/wp_query/#tag-parameters

Reply:
Just removed it. Was working on that on another branch and accidentally brought that over.